addons/l10n_it_edi/data/neutralize.sql

```diff
@@ -0,0 +1,3 @@
+-- neutralize Fattura Electronica (FatturaPA)
```

Comment:
Sorry for the nitpick, it's "Fattura Elettronica". 
If you took it from the Settings, there's a bugfix needed for that term also 🤣
I just wouldn't like the wrong name to spread.